data/kshitij9896.json

```
@@ -6,39 +6,51 @@
   "socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/kshitij_bansod" },
     { "icon": "FaGithub", "url": "https://github.com/kshitij9896" },
-    { "icon": "FaDev", "url": "https://dev.to/kshitij9896" },
-    { "icon": "FaBlog", "url": "https://www.showwcase.com/thisiskshitijbansod" }
+    { "icon": "FaDev", "url" : "https://dev.to/kshitij9896"},
+    { "icon": "FaBlog", "url" : "https://www.showwcase.com/thisiskshitijbansod"}
   ],
   "links": [
     {
-      "group": "Socials",
+      "group":"Services",
+      "name": "Can Help You With",
+      "url": "https://thisiskshitijbansod.netlify.app/",
+      "icon": "FaGlobe",
```

same here remove last comma  
```suggestion
      "icon": "FaGlobe"
```